cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -379,7 +379,7 @@ features:
         toggle:
           clearAll: Clear All
           selectAll: Select All
-      directivesTabLabel: Directives ({directivesCount})
+      directivesTabLabel: Transformation steps ({directivesCount})
```

Comment:
Will this actually fit in the tab without wrapping?

Reply:
Yes. It behaves the same way as "Columns" tab on resize.  Do you have suggestions for any changes?

![directivestab](https://user-images.githubusercontent.com/1452845/52303244-51e52f00-2944-11e9-9363-df5e0d441ac8.gif)

Reply:
👍 Talked to @leacd .